kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardTransactionDetail.xml

```diff
@@ -106,6 +106,11 @@ http://rice.kuali.org/dd http://rice.kuali.org/dd/dd.xsd">
 </bean>
 
 <!-- Attribute Definitions -->
+    <bean id="ProcurementCardTransactionDetail-transactionOriginalCurrencyAmount" parent="ProcurementCardTransactionDetail-transactionOriginalCurrencyAmount-parentBean">
+        <property name="validationPattern">
+            <bean parent="FloatingPointValidationPattern" p:allowNegative="true" />
+        </property>
```

Comment:
What is the proper way for validation for allowing negatives? I've seen this other variant in the kfs code:
<property name="validationPattern">
	<ref bean="FloatingPointAllowNegativeValidation" />
</property>